tests/test_embed_functionality.py

```diff
@@ -0,0 +1,35 @@
+from playwright.sync_api import Page, expect
+import os
+
+# Get the current working directory
+project_root = os.path.abspath(os.path.join(os.path.dirname(__file__), ".."))
+file_name = "www/index.html"
+full_path = "file://" + os.path.join(project_root, file_name)
+
```

Comment:
Let's make a python fixture for this (can probably copy the one from jncraton's pr earlier today) to keep consistency

Reply:
Alternative, this change new test could live in the [tests/main_page_tests.py](tests/main_page_tests.py) and use the same fixture as the other tests for this page.